src/packages/@ncigdc/utils/constants.js

```diff
@@ -222,3 +222,19 @@ export const IMPACT_SHORT_FORMS = {
     unknown: 'UN',
   },
 };
+
+export const FAKE_USER = localStorage.ALLOW_FAKE_USER
```

Comment:
this doesn't allow real downloads to happen because that requires a token like we discussed but could you load this key from process.env instead of localStorage so user doesn't have a chance to load fake user and scare themselves? :P

Reply:
Np.